src/AspNetCoreModuleV2/AspNetCore/src/applicationinfo.cpp

```diff
@@ -213,7 +207,7 @@ APPLICATION_INFO::EnsureApplicationCreated(
             // FindRequestHandlerAssembly obtains a global lock, but after releasing the lock,
             // there is a period where we could call
 
-            hr = FindRequestHandlerAssembly();
+            hr = FindRequestHandlerAssembly(&options);
```

Comment:
I don't like returning options from FindRequestHandlerAssembly, it is only used for the exeLocation.